ReSharper.FSharp/src/Daemon.FSharp/src/FSharpSymbolHighlightingUtil.cs

```diff
@@ -76,15 +79,15 @@ public static string GetMfvHighlightingAttributeId([NotNull] this FSharpMemberOr
       if (IsMangledOpName(mfv.LogicalName))
         return FSharpHighlightingAttributeIdsModule.Operator;
 
-      if (mfv.FullType.IsFunctionType || mfv.IsTypeFunction)
+      var fsType = mfv.FullType;
+      if (fsType.IsFunctionType || mfv.IsTypeFunction || fsType.IsAbbreviation && fsType.AbbreviatedType.IsFunctionType)
```

Comment:
`fsType.AbbreviatedType` can be another abbreviation, can't it? Would it be better to move checking `fsType` to a separate recursive function that'd go through all the abbreviations?

Reply:
Well the idea is using the fact that an abbreviation of a "int -> int" type is a FunctionType anyway. Like type T1 = int -> int; type T2 = T1. Here T2 is a function type so we don't need to go deeper to T1. So I felt that recursive check was redundant. I will do more of my own tests to make sure.

Reply:
This holds true even if we have chained types (t2 = t1, t3 = t2, t4 = t3 etc..). They are all function types anyway

Reply:
Great, thanks for checking/confirming! Could you add a test for it too, please?

Reply:
Sure. Added.